rpc/wrtc_call_queue_mongodb.go

```diff
@@ -876,6 +891,10 @@ func (queue *mongoDBWebRTCCallQueue) SendOfferInit(
 		return "", nil, nil, nil, err
 	}
 
+	// An offer initialization (after verifying the host queue size), indicates an attempted
+	// connection establishment attempt.
+	connectionEstablishments.Inc()
```

Comment:
Perhaps obviously, this metric will not capture connection establishment attempts _not_ made through app's signaling server (e.g. via direct gRPC or internal signaling). It will also not capture connection establishment attempts that did not reach app's signaling server due to network issues.

Finally, it will not capture any connection establishment attempt that returns early on L891 above. If I understand the logic correctly there, we return early when there are 50 active connection establishments to a single machine. I _could_ increment even in that case, and increment failures when we return early on L891 if that's desired.